metadata-ingestion/docs/sources/teradata/teradata_pre.md

```diff
@@ -5,24 +5,26 @@
     ```
 2. Create a user with the following privileges:
     ```sql
-    GRANT SELECT ON dbc.columns TO datahub;
-    GRANT SELECT ON dbc.databases TO datahub;
-    GRANT SELECT ON dbc.tables TO datahub;
-    GRANT SELECT ON DBC.All_RI_ChildrenV TO datahub;
-    GRANT SELECT ON DBC.ColumnsV TO datahub;
-    GRANT SELECT ON DBC.IndicesV TO datahub;
-    GRANT SELECT ON dbc.TableTextV TO datahub;
-    GRANT SELECT ON dbc.TablesV TO datahub;
-    GRANT SELECT ON dbc.dbqlogtbl TO datahub; -- if lineage or usage extraction is enabled
+GRANT SELECT ON DBC.DatabasesV TO datahub;
+GRANT SELECT ON DBC.TablesV TO datahub;
+GRANT SELECT ON DBC.ColumnsV TO datahub;
+GRANT SELECT ON DBC.IndicesV TO datahub;
+GRANT SELECT ON dbc.TableTextV TO datahub;
+GRANT SELECT ON DBC.All_RI_ChildrenV TO datahub;
+
+-- if lineage or usage extraction is enabled
+GRANT SELECT ON dbc.dbqlogtbl TO datahub; 
+GRANT SELECT ON dbc.QryLogV TO datahub;
+GRANT SELECT ON dbc.QryLogSqlV TO datahub;
```

Comment:
Looks like the removal of indentation causes issues in the markdown rendering

<img width="925" alt="image" src="https://github.com/user-attachments/assets/4d11ddca-6b93-4542-8daf-41f14f1ea00e">